api/blueprints/routes.py

```diff
@@ -56,13 +56,17 @@ def get_movie(movie_id):
         'genres': [g.name for g in movie.genres],
     }
     data.update(related_data)
-    return jsonify({'movie': data})
+    return jsonify({'results': data})
 
 
 @bp.route(f'{API_BASE_PATH}/movies', methods=['GET'])
 def get_movies():
-    page = request.args.get('page', 1, type=int)
-    movies = Movie.query.paginate(page, per_page=10).items
+    limit = request.args.get('limit', 5, type=int)
+    start = request.args.get('start', 0, type=int)
+    page_number = int(start / limit) + 1
+    paginated = Movie.query.paginate(page=page_number, per_page=limit)
+
+    movies = paginated.items
     movies_data = {}
     for m in movies:
```

Comment:
I think it would make sense to return a `List` as `results` here. It's uncommon to respond with a `<id:dict>` map for multiple results as a list conveys an order while a map does not.